lightning_pose/models/backbones/torchvision.py

```diff
@@ -0,0 +1,118 @@
+import torch
+import torchvision.models as tvmodels
+from typeguard import typechecked
+
+
+@typechecked
+def build_backbone(
+    backbone_arch: str, pretrained: bool = True, model_type: str = "heatmap", **kwargs,
```

Comment:
the `pretrained` argument is deprecated for `torchvision.models`? we get tons of warnings.
we should use `weights` instead. checkout the new neat guidelines here; it'll enable loading different pretrained weights for the same resnet50 archi.
https://pytorch.org/vision/stable/models.html

Reply:
thanks for pointing this out, updated now